components/Stats.js

```diff
@@ -1,7 +1,27 @@
 import React from "react";
+import { Row, Col, Container } from "react-bootstrap";
 
 const Stats = () => {
-	return <div>Stats</div>;
+	return (
+		<div>
+			<Container>
+				<Row className='font-lexend text-4xl decoration-black font-semibold'>
+					<Col xs={4} md={3} className='flex justify-center'>
```

Comment:
Regarding the text going onto 2 lines, you can prevent that by adding `whitespace-nowrap` to force it to be on one line